source/implementations/linux/Meadow.Linux/Hardware/Pinouts/RaspberryPi5.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Meadow.Hardware;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Meadow.Pinouts
+{
+    public class RaspberryPi5 : RaspberryPi
+    {
+        public RaspberryPi5()
+        {
+            // the sysfs base for the pinctrl-rpi1 is 53
+            // the gpiochip bank for pinctrl-rpi1 on rpi5 is gpiochip4
+            foreach (LinuxFlexiPin pin in this.AllPins) {
+                pin.SysFsGpio += 53;
```

Comment:
Does this apply to `GpiodOffset` also?

Reply:
hey @bcr , no for libgpiod what's changed is the gpiochip number, but the offset continues the same.

Reply:
Just to make sure -- there's an integer property called `SysFsGpio` that you have added 53 to. There is an additional integer property called `GpiodOffset` that in the base class has the exact same value as `SysFsGpio` but is not adjusted here. I do understand that you changed the `GpiodChip` field also.